eos/b-decays/b-to-psd-l-nu.cc

```diff
@@ -419,7 +515,7 @@ namespace eos
     }
 
     double
-    BToDLeptonNeutrino::differential_r_d(const double & s) const
+    BToPseudoscalarLeptonNeutrino::differential_r_d(const double & s) const
```

Comment:
See also comment in header file: rename "..._r_d" -> "..._r_P"

Reply:
Also resolved via ObservableRatio.